test/integration/storage_versions_test.go

```diff
@@ -23,21 +24,12 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// TODO: enable once storage is separable
-// func TestStorageVersionsSeparated(t *testing.T) {
-// 	runStorageTest(t, "separated",
-// 		autoscaling_v1.SchemeGroupVersion,
-// 		batch_v1.SchemeGroupVersion,
-// 		extensions_v1beta1.SchemeGroupVersion,
-// 	)
-// }
-
-func TestStorageVersionsUnified(t *testing.T) {
+func TestStorageVersions(t *testing.T) {
```

Comment:
do we have a test to make sure that an extensions/v1beta1 Job can be read from storage, and if updated, gets persisted back as a batch/v1 Job?